src/Sylius/Bundle/ImportExportBundle/Behat/ImportExportContext.php

```diff
@@ -0,0 +1,673 @@
+     * @And there are following export profiles configured:
+     */
+    public function thereAreExportProfiles(TableNode $table)
+    {
+        $manager = $this->getEntityManager();
+        $repository = $this->getRepository('export_profile');
+
+        foreach ($table->getHash() as $data) {
+            $this->thereIsExportProfile(
+                $data['name'],
+                $data['description'],
+                $data["code"],
+                $data['reader'],
+                $data['reader_configuration'],
+                $data['writer'],
+                $data['writer_configuration'],
+                false
+            );
+        }
+
+        $manager->flush();
+    }
+
+    /**
+     * @Given there are following import profiles configured:
+     * @And there are following import profiles configured:
+     */
+    public function thereAreImportProfiles(TableNode $table)
+    {
+        $manager = $this->getEntityManager();
+
+        foreach ($table->getHash() as $data) {
+            $this->thereIsImportProfile(
+                $data['name'],
+                $data['description'],
+                $data["code"],
+                $data['reader'],
+                $data['reader configuration'],
+                $data['writer'],
+                $data['writer configuration'],
+                false
+            );
+        }
+
+        $manager->flush();
+    }
+
+    /**
+     * @Given there are following export jobs set:
+     * @And there are following export jobs set:
+     */
+    public function thereAreExportJobs(TableNode $table)
+    {
+        $manager = $this->getEntityManager();
+
+        foreach ($table->getHash() as $data) {
+            $this->thereIsExportJob($data['status'], $data['start_time'], $data["end_time"], $data['created_at'], $data['updated_at'], $data["export_profile"], false);
+        }
+
+        $manager->flush();
+    }
+
+    /**
+     * @Then I should be on the export jobs index page for profile with code :code
+     */
+    public function iShouldBeOnTheExportJobsIndexPageForProfileWithCode($code)
+    {
+        $exportProfile = $this->findOneBy('export_profile', array('code' => $code));
+
+        $this->assertSession()->addressEquals($this->generatePageUrl('sylius_backend_export_job_index', array('profileId' => $exportProfile->getId())));
+    }
+
+    /**
+     * @Given I am on the export jobs index page for profile with code :code
+     */
+    public function iAmOnTheExportJobsIndexPageForProfileWithCode($code)
+    {
+        $exportProfile = $this->findOneBy('export_profile', array('code' => $code));
+
+        $this->getSession()->visit($this->generatePageUrl('sylius_backend_export_job_index', array('profileId' => $exportProfile->getId())));
+    }
+
+    /**
+     * @When I press :button near :number export job
+     * @When I click :button near :number export job
+     */
+    public function iPressNearExportJob($button, $number)
```

Comment:
This method has the same logic as https://github.com/michalmarcinkowski/Sylius/pull/5/files#diff-bde44e796dc0eb506abf50be9392bbb2R320. You can add both adnotations here.